### DISCUSSION TITLE
Split input by comma

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#36571] [Actions] [answered]
I want to run the following `bash` step

```
for i in ${variable//,/ }
do
    echo "$i"
done
```

however in the place of `variable` I want to pass an input.

Is this feasible?

**Comment by Nishu0** [comment]
@pantelis-karamolegkos 
Try this below code!
```
read variable
for i in ${variable//,/ }
do
    echo "$i"
    read input
    echo $input;
done
```

**Comment by pantelis-karamolegkos** [comment]
@Nishu0 thanks but I was referring to a github actions' input, as in `${{ inputs.my-input }}`

**Comment by ChristopherHX** [comment, marked as answer]
This could work, if your question was how to pass variables to bash. Both local and env bash variables are accesed the same way, if I'm correct.
```yaml
- run: |
    for i in ${variable//,/ }
    do
        echo "$i"
    done
  env:
    variable: ${{ inputs.my-input }}
```
